packages/usa-tooltip/src/index.js

```diff
@@ -356,6 +357,10 @@ const setUpAttributes = (tooltipTrigger) => {
   // place the text in the tooltip
   tooltipBody.textContent = tooltipContent;
 
+  // insert aria span before text content to be read first
+  ariaSpan.setAttribute("aria-label", "Tooltip :");
```

Comment:
iirc when @amycole501 and I chatted about the wording _I think_ she said "colon" gets read out by jaws or nvda? I do not hear it either since I'm also on VO. I don't see an issue with removing but Amy might have input when she gets back.

Reply:
It's common for screen readers outside of JAWS to not announce the colon. I would be curious to hear what the tooltip sounds like without the colon, as the colon seems out of place with JAWS. With NVDA, it generates a pause between "tooltip" and the content. I think that interaction is ideal in this situation, as I do not think the user needs to hear "colon".  It seems disrupt the flow of JAWS, since there is no pause. I fear that could be overwhelming to some users. Is there a way we could treat it like a period, and provide some of pause between the words "tooltip" and the associated content? @mlloydbixal

Reply:
@alex-hull I removed the colon and gave the text content it's own span so it starts on a new line. That _should_ cause the screen reader to pause. Can you test with JAWS and NVDA? Thanks for the suggestion!

Reply:
Listening in both, now there is no pause for either NVDA or JAWS. It moves straight into the content. [Deque](https://dequeuniversity.com/library/aria/tooltip) is the only accessibility option that I see using a colon. But again, I don't think that is needed.

Reply:
Listened to this one more time, and it still sounds like there is no pause. I say we continue to investigate it in the future, but just keep the colon out for now. Looking at a few more examples ([A11y style guide](https://a11y-style-guide.com/style-guide/section-forms.html#kssref-forms-tooltips), [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role)), it does not look like the colon (or other punctuation) is commonly placed in the tooltip role. I thinking having it there would provide a not great experience for JAWS users. So proceed on! @mlloydbixal